tests/console/postgresql96server.pm

```diff
@@ -27,7 +27,6 @@ sub run {
 
     # check the status
     assert_script_run 'systemctl show -p ActiveState postgresql.service | grep ActiveState=active';
-    assert_script_run 'systemctl show -p SubState postgresql.service | grep SubState=running';
```

Comment:
hm, I'm not convinced. I checked myself as described in https://bugzilla.suse.com/show_bug.cgi?id=1060639#c2 and the state is 'active (exited)' even though the postgres process is running. So maybe there *is* something wrong with the service. Maybe the first step would be to add the test module also to openSUSE Tumbleweed tests.

Reply:
Glad I did not convince you: https://bugzilla.suse.com/show_bug.cgi?id=1060639#c9 :).

Reply:
Commit dropped.